### PR TITLE
Added local deployment option to run OC cluster on host machine

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: ansible/reset_local_environment.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    #- demo_prep
+  post_tasks:
+    - set_fact:
+        msg: |
+            Hostname:                  {{ openshift_hostname }}
+            Next steps:
+            1) Visit https://apiserver-service-catalog.{{ openshift_hostname }}
+            2) Accept the certificate
+            3) Visit https://{{ openshift_hostname }}:8443 for the console
+            4) Provision a RDS APB into awsdemo project
```
Comment (jwmatthews) [inline ansible/reset_local_environment.yml]: Let's remove step #4 - #7 as we aren't doing the RDS demo for local.

File: ansible/roles/ansible_service_broker_setup/defaults/main.yml [hunk 2]
```diff
@@ -1 +1,2 @@
-go_path: /root/go
+#go_path: /root/go
```
Comment (jwmatthews) [inline ansible/roles/ansible_service_broker_setup/defaults/main.yml]: Remove the commented out code

File: ansible/roles/ansible_service_broker_setup/tasks/main.yml [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
-  - name:
-    file:
-      path: "{{ go_path }}/src/github.com/fusor"
-      state: directory
-      mode: 0755
+  #- name:
+  #  file:
+  #    path: "{{ go_path }}/src/github.com/fusor"
+  #    state: directory
+  #    mode: 0755
```
Comment (jwmatthews) [inline ansible/roles/ansible_service_broker_setup/tasks/main.yml]: Remove the commented out code

File: ansible/roles/openshift_setup/tasks/main.yml [hunk 4]
```diff
@@ -1,53 +1,94 @@
...
+    when: openshift_release_ext == "tar.gz"
+
+  #- name: Install oadm
+  #  copy:
+  #    remote_src: True
+  #    src: /tmp/{{ openshift_server_release_ver }}/oadm
+  #    dest: /usr/local/bin/oadm
+  #    owner: root
+  #    group: root
+  #    mode: 0755
```
Comment (jwmatthews) [inline ansible/roles/openshift_setup/tasks/main.yml]: Remove the commented out code

File: ansible/setup_local_environment.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    #- demo_prep
+  post_tasks:
+    - set_fact:
+        msg: |
+            Hostname:                  {{ openshift_hostname }}
+            Next steps:
+            1) Visit https://apiserver-service-catalog.{{ openshift_hostname }}
+            2) Accept the certificate
+            3) Visit https://{{ openshift_hostname }}:8443 for the console
+            4) Provision a RDS APB into awsdemo project
```
Comment (jwmatthews) [inline ansible/setup_local_environment.yml]: Remove steps #4-7 similar to reset_env

File: local/common_vars [hunk 6]
```diff
@@ -0,0 +1,9 @@
+AWS_SSH_KEY_NAME="${AWS_SSH_KEY_NAME:-splice}"
+export AWS_SSH_KEY_NAME
+
+export CLUSTER_IP="$(ifconfig docker0 | grep inet | awk '{print $2}' | head -1)"
+#export CLUSTER_IP="172.17.0.1"
```
Comment (jwmatthews) [inline local/common_vars]: Remove the commented out code

Comment (jwmatthews) [inline local/common_vars]: Should we add another variable in EXTRA_VARS to ensure we don't attempt to create lets_encrypt certs for local running?

File: local/reset_environment.sh [hunk 7]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+[ -z "$AWS_ACCESS_KEY_ID" ] && echo "Missing environment variable:  AWS_ACCESS_KEY_ID" && exit 1;
```
Comment (jwmatthews) [inline local/reset_environment.sh]: How about we remove the AWS Env Var checks for running local?

File: local/run_setup_local.sh [hunk 8]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+[ -z "$AWS_ACCESS_KEY_ID" ] && echo "Missing environment variable:  AWS_ACCESS_KEY_ID" && exit 1;
+[ -z "$AWS_SECRET_ACCESS_KEY" ] && echo "Missing environment variable:  AWS_SECERT_ACCESS_KEY" && exit 1;
+[ -z "$AWS_SSH_PRIV_KEY_PATH" ] && echo "Missing environment variable:  AWS_SSH_PRIV_KEY_PATH\nPlease set this to the path for your SSH private key\n" && exit 1;
```
Comment (jwmatthews) [inline local/run_setup_local.sh]: How about we remove the AWS Env Var checks for running local?

File: local/common_vars (hunk 6 above)
Comment (dymurray) [inline local/common_vars]: In local/common_vars there is an EXTRA_VARS argument to have 'use_ssl' be false

**Comment by jwmatthews** [comment]
Looks good!

**Pull request merged**